README.md

```diff
@@ -47,6 +47,10 @@ are instances of such transformations. Others are
 [`pmap`](#spmd-programming-with-pmap) for single-program multiple-data (SPMD)
 parallel programming of multiple accelerators, with more to come.
 
+**Note**: Try `shard_map` (a.k.a. `shmap`) - a new more flexible API that supersedes
+`jax.pmap`. Check out the `jax.experimental.shard_map`
+[JAX Enhancement Proposal (JEP) to learn more](https://jax.readthedocs.io/en/latest/jep/14273-shard-map.html).
```

Comment:
I don't think this link is appropriate for the README – it's more of a design doc than a user guide. (I believe @mattjj is working on a `shard_map` user guide, but it's not in yet)